OrbitCore/Pdb.h

```diff
@@ -231,9 +235,9 @@ class Pdb {
   std::map<std::string, std::vector<std::string> > m_RegFunctionsMap;
 
   // Data
-  std::string m_Name;
-  std::string m_FileName;
-  std::string m_LoadedModuleName;
+  std::string m_Name;              // name of the file containing the symbols
+  std::string m_FileName;          // full path of file containing the symbols
+  std::string m_LoadedModuleName;  // full path of the module
```

Comment:
Same comment as before: If comments are necessary, maybe it's time to change the variable names?

Reply:
Same as before, will do in different PR